pytorch_lightning/callbacks/quantization.py

```diff
@@ -126,9 +134,23 @@ def custom_trigger_last(trainer):
         quantize_on_fit_end: perform the quantization in `on_fit_end`.
             Note that once converted, the model cannot be put in training mode again.
 
+        disable_observers: disable fake-quantization modules' observers during the provided stages:
+
+            - ``'train'``: the observers will be disabled during training.
+            - ``'validate'``: the observers will be disabled during validating.
+              Note that we don't disable observers during the sanity check as the model hasn't been calibrated with
+              training data yet. After the sanity check, the fake-quantization modules are restored to initial states.
+            - ``'test'``: the observers will be disabled during testing.
+            - ``'predict'``: the observers will be disabled during predicting.
+
+            Note that we only handle observers belonging to fake-quantization modules. When ``qconfig`` is a ``str`` and
+            ``observer_type`` is ``'histogram'``, the observers won't belong to any fake-quantization modules and will
+            not be disabled.
+
     """
 
     OBSERVER_TYPES = ("histogram", "average")
+    ALL_DISABLE_OBSERVER_STAGES = ("train", "validate", "test", "predict")
```

Comment:
is the ALL prefix necessary? we don't have it for the other constant

Reply:
Should we use

1. names without `ALL` prefixes:
```python
OBSERVER_TYPES = ...
DISABLE_OBSERVER_STAGES = ...
```

or as @Borda suggested

2. names with `ALL` prefixes: (change `OBSERVER_TYPES` to `ALL_OBSERVER_TYPES` in this PR)
```python
ALL_OBSERVER_TYPES = ...
ALL_DISABLE_OBSERVER_STAGES = ...
```
3. only `ALL_DISABLE_OBSERVER_STAGES` with the `ALL` prefix
```python
OBSERVER_TYPES = ...
ALL_DISABLE_OBSERVER_STAGES = ...
```

Reply:
1 or 2, but not 3. 3 is not consistent.

Reply:
It seems that changing `OBSERVER_TYPES` may break users' code. The `ALL_DISABLE_OBSERVER_STAGES` is reverted to `DISABLE_OBSERVER_STAGES`.